### PR TITLE
Release 1.10.0

### PR DESCRIPTION
**Pull request by minevala**
Release 1.10.0

### CODE REVIEW

**Review by ronijaakkola — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,62 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+- [Icons] Thumb icons
+
+#### Changed
+
+- [Tabs] Add support to use Tabs child components in Tabs namespace, for example, Tabs.TabList
+
+#### Fixed
+
+- [Text Input, TextArea] Prevent rounded corners in iOS browsers
```
Comment (VilleMiekkoja) [inline CHANGELOG.md:33]: Should it be Text Area, as Text Input is also written as two separate words?

File: site/docs/about/roadmap.mdx [hunk 2]
```diff
@@ -53,29 +53,45 @@ For previously released features, see changelogs in the [What is new section](/a
...
+~~- Update: Koros improvements~~<br/>
+~~- Update: Smaller improvements and fixes to multiple components~~<br/>
 
+
+**Q1 2022**
 - Pattern: Multi-step forms
+- Component: Form stepper
+
+
 - Pattern: Cookie consent
```
Comment (VilleMiekkoja) [inline site/docs/about/roadmap.mdx:67]: Should we list Cookie consent also as component here?

Comment (minevala) [reply site/docs/about/roadmap.mdx:67]: Yes, probably

**Review by VilleMiekkoja — COMMENTED** [review]
Nice looking! Added couple minor comments

File: CHANGELOG.md (hunk 1 above)
Comment (minevala) [reply CHANGELOG.md:33]: I used the same convention as in elsewhere 🤔

File: site/docs/about/roadmap.mdx (hunk 2 above)
Comment (minevala) [reply site/docs/about/roadmap.mdx:67]: - [ ] add Cookie Consent component in the roadmap

**Review by VilleMiekkoja — APPROVED** [review]
🎖️

File: CHANGELOG.md (hunk 1 above)
Comment (minevala) [reply CHANGELOG.md:33]: There seems to three kind of convention for these two: "Time input", "Time Input", "TimeInput" and "Text area", "TextArea" and "Textarea" 🙈

Comment (ronijaakkola) [reply CHANGELOG.md:33]: TextInput or Text input. TextInput when talking about the specific component and "Text input" when talking about the element in general.

Comment (minevala) [reply CHANGELOG.md:33]: - [ ] Fix component naming

**Pull request merged**